ext/crates/fp/src/prime.rs

```diff
@@ -506,11 +507,18 @@ impl Iterator for BinomialIterator {
 #[cfg(test)]
 mod tests {
     use super::*;
+    #[cfg(feature = "odd-primes")]
     use crate::constants::PRIMES;
 
     #[test]
     fn inverse_test() {
-        for &p in PRIMES.iter() {
+        #[cfg(feature = "odd-primes")]
+        let primes = PRIMES;
```

Comment:
Can PRIMES be [2] when odd-primes is false? (And maybe similarly have a TEST_PRIMES that is either [2] or [2, 3, 5, 7, 11])

Reply:
Actually I realized that most problems are fixed if we just let `num_primes = 1` in build.rs when odd-primes is disabled. Thanks for pointing that out